src/components/Class/index.js

```diff
@@ -124,9 +124,18 @@ function Class() {
               type="email"
               name="facilitator_email"
               id="facilitator_email"
-              pattern="[a-zA-Z0-9.-_]{1,}@[a-zA-Z.-]{2,}[.]{1}[a-zA-Z]{2,}"
               required
               aria-required
+              onChange={(e) => {
+                let pattern =
```

Comment:
It is not rendering the error according to the situation. Like if I am giving the correct email id.Still it is showing an error because of setting event. target.setCustomValidity('Your message'). That's why I used two situations if the pattern does not match the email; then only it should show an error. Otherwise, it proceeds.